raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -735,88 +703,6 @@ def test_events_for_balanceproof_lock_expired():
     assert not events
 
 
-def test_regression_onchain_secret_reveal_must_update_channel_state():
```

Comment:
Why was this test removed?

Reply:
It was moved to test_mediatorstate_regression.py